src/main/java/io/orangebeard/listener/OrangebeardTestSystemListener.java

```diff
@@ -187,32 +191,36 @@ private UUID getAndOrStartSuite(TestPage testPage) {
             suitePath = suitePath + "." + suite;
             suiteId = runContext.getSuiteId(suitePath);
             if (suiteId == null) {
+                String description = null;
                 Set<Attribute> suiteAttrs = null;
-                if(suitePath.endsWith(fullSuiteName)) {
-                     suiteAttrs = retrieveSuiteMetaData(testPage);
+                PageData suitePageData = getPageDataForSuite(suitePath, testPage.getSourcePage());
+                if (suitePageData != null && suitePageData.getAttribute(WikiPageProperty.SUITES) != null) {
+                    suiteAttrs = extractTags(suitePageData.getAttribute(WikiPageProperty.SUITES));
```

Comment:
extractAttributes ipv extractTags? ik heb er in de backend ook overal attributes van gemaakt :-).